apis/keda/v1alpha1/scaledobject_webhook_test.go

```diff
@@ -602,40 +579,16 @@ var _ = It("should validate the so update if it's removing the finalizer even if
 	err = k8sClient.Create(context.Background(), workload)
 	Expect(err).ToNot(HaveOccurred())
 
-	Eventually(func() error {
-		return k8sClient.Create(context.Background(), so)
-	}).ShouldNot(HaveOccurred())
+	err = k8sClient.Create(context.Background(), so)
+	Expect(err).ToNot(HaveOccurred())
 
 	workload.Spec.Template.Spec.Containers[0].Resources.Requests = nil
 	err = k8sClient.Update(context.Background(), workload)
 	Expect(err).ToNot(HaveOccurred())
 
 	so.ObjectMeta.Finalizers = []string{}
-	Eventually(func() error {
-		return k8sClient.Update(context.Background(), so)
-	}).ShouldNot(HaveOccurred())
-})
-
-var _ = It("shouldn't create so when stabilizationWindowSeconds exceeds 3600", func() {
```

Comment:
Validation for `stabilizationWindowSeconds` was added in https://github.com/kedacore/keda/pull/4983 via a validation patch directly on the CRD. It looks like the patch doesn't get picked up by the admission webhook, so the webhook actually thinks this is a valid object. This test is a false positive, and I don't think it belongs in this file with the rest of the admission webhook unit tests

```
# error message when rejected by admission webhook
Error from server (Forbidden): error when creating "scaledobject.yaml": admission webhook "vscaledobject.kb.io" denied the request: type is cpu , but fallback it is not supported by the CPU & memory scalers

# error message when rejected by CRD validation
The ScaledObject "keda-test" is invalid: spec.advanced.horizontalPodAutoscalerConfig.behavior.scaleDown.stabilizationWindowSeconds: Invalid value: 36000: spec.advanced.horizontalPodAutoscalerConfig.behavior.scaleDown.stabilizationWindowSeconds in body should be less than or equal to 3600
```

I'm not sure how to test a direct CRD patch though so I haven't added a replacement test - need to look into it more, or open to suggestions here.